docs/cudf/source/basics.rst

```diff
@@ -0,0 +1,40 @@
+| Kind of Data           | Data Type  | Scalar                                                                              | String Aliases                              |
++========================+============+=====================================================================================+=============================================+
+| Integer                |            | `int <https://docs.python.org/3/library/functions.html#int>`_                       | ``'int8'``, ``'int16'``, ``'int32'``,       |
+|                        |            |                                                                                     | ``'int64'``, ``'uint8'``, ``'uint16'``,     |
+|                        |            |                                                                                     | ``'uint32'``, ``'uint64'``                  |
++------------------------+------------+-------------------------------------------------------------------------------------+---------------------------------------------+
+| Float                  |            | `float <https://docs.python.org/3/library/functions.html#float>`_                   | ``'float32'``, ``'float64'``                |
++------------------------+------------+-------------------------------------------------------------------------------------+---------------------------------------------+
+| Strings                |            | `str <https://docs.python.org/3/library/stdtypes.html#str>`_                        | ``'string'``, ``'object'``                  |
++------------------------+------------+-------------------------------------------------------------------------------------+---------------------------------------------+
+| Datetime               |            | `numpy.datetime64 <https://numpy.org/doc/stable/reference/arrays.datetime.html>`_   | ``'datetime64[s]'``, ``'datetime64[ms]'``,  |
+|                        |            |                                                                                     | ``'datetime64[us]'``, ``'datetime64[ns]'``  |
++------------------------+------------+-------------------------------------------------------------------------------------+---------------------------------------------+
+| Timedelta              |            | `numpy.timedelta64 <https://numpy.org/doc/stable/reference/arrays.datetime.html>`_  | ``'timedelta64[s]'``, ``'timedelta64[ms]'``,|
+| (duration type)        |            |                                                                                     | ``'timedelta64[us]'``, ``'timedelta64[ns]'``|
++------------------------+------------+-------------------------------------------------------------------------------------+---------------------------------------------+
+| Categorical            |            | (none)                                                                              | ``'category'``                              |
```

Comment:
Should we have `CategoricalDtype` in here since we technically have a separate dtype from Pandas?